frontend/ui/elements/reader_menu_order.lua

```diff
@@ -45,7 +44,9 @@ local order = {
         "screen",
         "screensaver",
         "document",
+        "----------------------------",
```

Comment:
Can we move that separator just above, between `screensaver` and `document`?
`navigation` would appear only on devices with keys, and we would get a section with `document > end of document action` and `follow links >`, which feels a bit more related to each other than to `screensaver` and `language`.

Reply:
Sure, I'll do that.